tests/infra/consortium.py

```diff
@@ -875,3 +875,10 @@ def check_for_service(self, remote_node, status, recovery_count=None):
                 assert (
                     recovery_count is None or current_recovery_count == recovery_count
                 ), f"Current recovery count {current_recovery_count} is not expected {recovery_count}"
+
+    def remove_old_jwt_tables(self, remote_node, issuer):
```

Comment:
Looks like we're not actually calling this anywhere. We should add a call to this in the `lts_compatibility` test (or one of the tests in `recovery.py` that recovers from an old ledger), to fully confirm that we can have a service where these tables are present _before_ the call, removed _after_ the call, and not repopulated if we do a jwt refresh.